src/backend/cdb/cdbfts.c

```diff
@@ -108,6 +108,10 @@ FtsNotifyProber(void)
 	int32			started;
 	int32			done;
 
+	/* Ignore if we don't have a FTS probe process, like a standby QD in a mirrored cluster. */
+	if (FtsProbePID() == 0)
+		return;
+
```

Comment:
Have you considered placing this check into the infinite loop below? Could it be better that way?

Reply:
It is already done at this place in GPDB 7x. And I think earlier we get out of the function in case there is no FTS - better. So I prefer to leave it here.